pkg/koordlet/runtimehooks/hooks/groupidentity/interceptor.go

```diff
@@ -17,14 +17,39 @@ limitations under the License.
 package groupidentity
 
 import (
+	"encoding/json"
+	"fmt"
+
 	"k8s.io/klog/v2"
 	"k8s.io/utils/pointer"
 
 	ext "github.com/koordinator-sh/koordinator/apis/extension"
+	slov1alpha1 "github.com/koordinator-sh/koordinator/apis/slo/v1alpha1"
 	"github.com/koordinator-sh/koordinator/pkg/koordlet/runtimehooks/protocol"
 	"github.com/koordinator-sh/koordinator/pkg/koordlet/util"
 )
 
+func getPodCPUQoSConfig(annotations map[string]string) (*slov1alpha1.CPUQOSCfg, error) {
```

Comment:
better to mv this func to api/slo/v1aplha1/pod.go

Reply:
this function is different to exist functions like `GetPodMemoryQoSConfig`, which receive parameter from annotation, so I define it in local file. If we move it to `apis/slo/v1aplha1/pod.go`, how about named `GetPodCPUQoSConfigFromAnnotations`?

Reply:
or `GetPodCPUQoSConfigByAttrs`?